agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -179,7 +179,7 @@ function install_k8s_agent {
         # Pull the cluster name using the cluster ID using ibmcloud ks
         # since the current-context is not a user-friendly value
         fail=0
-        CLUSTER_NAME=$(ibmcloud ks cluster get --cluster "$IKS_CLUSTER_ID" --json | jq -r .name) || { fail=1 && echo "Failed to get the cluster name"; }
+        CLUSTER_NAME=$(kubectl get cm -n kube-system cluster-info -o yaml | grep ' "name": ' | cut -d'"' -f4) || { fail=1 && echo "Failed to get the cluster name"; }
```

Comment:
using jq can be cleaner
```
kubectl get cm -n kube-system cluster-info -o json -o jsonpath="{.data.cluster-config\.json}" | jq -r '.cluster_name'
```

This assumes that the current context is the correct one. I don't understand why this line is needed when this line is even cleaner:

https://github.com/draios/sysdig-cloud-scripts/blob/5e293e722cd0bec968e66c37ea7d18bcd709f2ee/agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh#L187